rholang/src/test/scala/rholang/rosette/CompilerTests.scala

```diff
@@ -3,49 +3,47 @@ package rholang.rosette
 import java.io.FileReader
 import java.nio.file.{Files, Path, Paths}
 
-import monix.execution.Scheduler.Implicits.global
-import coop.rchain.catscontrib.Capture._
 import coop.rchain.rholang.interpreter.{Interpreter, Runtime}
-import org.scalatest.FunSuite
+import monix.execution.Scheduler.Implicits.global
+import org.scalatest.{FunSuite, Matchers}
 
 import scala.collection.JavaConverters._
 import scala.concurrent.Await
 import scala.concurrent.duration._
-import scala.util.Try
 
-class CompilerTests extends FunSuite {
-  val mapSize     = 1024L * 1024L * 1024L
+class CompilerTests extends FunSuite with Matchers {
+  val mapSize     = 1024L * 1024L * 10
```

Comment:
@birchmd I've decreased the memory allocation from `10GB` to 10MB. I think this should be enough for this test.